### ISSUE TITLE
Bad interaction with `babel-preset-es2015`

### ISSUE THREAD

**Issue by anmonteiro** [airbnb/babel-plugin-dynamic-import-node#15]
Adding `babel-preset-es2015` along with `babel-plugin-dynamic-import-node` causes a bad interaction between the two that causes a failure in the compilation.

I think this may also have something to do with what the reporter is seeing in #13.

Here's the full stacktrace:

```
TypeError: unknown: Property value expected type of string but got null
      at Object.validate (node_modules/babel-types/lib/definitions/index.js:161:13)
      at validate (node_modules/babel-types/lib/index.js:505:9)
      at Object.builder (node_modules/babel-types/lib/index.js:466:7)
      at PluginPass.TemplateLiteral (node_modules/babel-plugin-transform-es2015-template-literals/lib/index.js:81:24)
      at newFn (node_modules/babel-traverse/lib/visitors.js:276:21)
      at NodePath._call (node_modules/babel-traverse/lib/path/context.js:76:18)
      at NodePath.call (node_modules/babel-traverse/lib/path/context.js:48:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:105:12)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitMultiple (node_modules/babel-traverse/lib/context.js:103:17)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:190:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitSingle (node_modules/babel-traverse/lib/context.js:108:19)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:192:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitMultiple (node_modules/babel-traverse/lib/context.js:103:17)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:190:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitSingle (node_modules/babel-traverse/lib/context.js:108:19)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:192:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitMultiple (node_modules/babel-traverse/lib/context.js:103:17)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:190:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:156:21)
      at TraversalContext.visitSingle (node_modules/babel-traverse/lib/context.js:108:19)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:192:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitMultiple (node_modules/babel-traverse/lib/context.js:103:17)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:190:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:115:19)
      at TraversalContext.visitQueue (node_modules/babel-traverse/lib/context.js:150:16)
      at TraversalContext.visitSingle (node_modules/babel-traverse/lib/context.js:108:19)
      at TraversalContext.visit (node_modules/babel-traverse/lib/context.js:192:19)
      at Function.traverse.node (node_modules/babel-traverse/lib/index.js:114:17)
      at traverse (node_modules/babel-traverse/lib/index.js:79:12)
      at File.transform (node_modules/babel-core/lib/transformation/file/index.js:548:35)
      at node_modules/babel-core/lib/transformation/pipeline.js:50:19
      at File.wrap (node_modules/babel-core/lib/transformation/file/index.js:564:16)
      at Pipeline.transform (node_modules/babel-core/lib/transformation/pipeline.js:47:17)
      at testPlugin (test/testPlugin.js:4:18)
      at Context.<anonymous> (test/index.js:18:22)
```

**Comment by ljharb** [comment]
Can you provide your babel config?

**Comment by anmonteiro** [comment]
@ljharb you can make the tests fail in this repo with this commit: https://github.com/anmonteiro/babel-plugin-dynamic-import-node/commit/2d3031088ebcac9a34f0a88474a9223e1432cde1

**Comment by ljharb** [comment]
Thanks, that helps a lot.

I've put up #16 to fix this (turns out this is what #12 was fixing too)

**Comment by anmonteiro** [comment]
Thanks!
